RELEASE.md

```diff
@@ -426,6 +427,14 @@ the size restrictions for publishing on PyPI so the default version that is publ
 These special support cases will be handled on a case by case basis and support may be continued if current PyTorch maintainers feel as though there may still be a
 need to support these particular versions of software.
 
+## Operating Systems
+
+PyTorch Linux x86_64 wheels are manylinux2014 compatible, i.e. they should be runnable on any Linux system with glibc-2.17 or above.
```

Comment:
Can we make  it as table ?
OS | Supported Architectures | Details
Linux x86 | CPU, CUDA  | Details about Linux x86
Linux arm64 | CPU, CUDA (Experimental) | Details about Linux arm64 builds 
MacOS x86 | CPU | Starting from 2.3 MacOS is x86 binaries are not officially supported
MacOS arm64 | CPU |
......

Reply:
Sounds good to me, though I would not mention accelerator here, as technically it should have no affect on supported OS matrix.